src/features/CareViewPatientsSummary/components/PatientDetailsCell.jsx

```diff
@@ -116,12 +117,13 @@ export const PatientDetailsCell = ({
           <span>{person.gender}</span>)<span className={"separator"}>|</span>
           <span>{person.age}</span>
           <FormattedMessage id={"AGE_YEARS_LABEL"} defaultMessage={"yrs"} />
-          {newTreatments > 0 && (
+          {(newTreatments > 0 || previousShiftPendingTasks.length > 0) && (
             <>
               <div className="treatments-notification">
-                <WarningIcon />
-                <span className="treatments-notification-span">
-                  {newTreatments + " New treatment(s): "}
+                <div className="warning_icon">
+                  <WarningIcon /> </div>
```

Comment:
We have an icon in carbon: WarningAlt
Usage: https://github.com/Bahmni/openmrs-module-ipd-frontend/blob/a2f92227c5956843a47507398c3a63af23c4e0d2/src/features/CareViewPatients/components/CareViewPatients.jsx#L389